src/components/usercomponents/Button.tsx

```diff
@@ -0,0 +1,23 @@
+
+interface ButtonEl{
+    type: any;
+    title: string;
+    children:string;
+    onClick?: () => void;
+    className?:string;
+}
+
+const Button = ({ type,title, children,className,onClick}:ButtonEl) => {
+  return (
+    <button 
+    type={type} 
+    title={title}
+    className={className || `bg-black rounded-[8px] w-[60%] text-white border border-[#E0E0E0] py-2 px-4 box-border hover:opacity-90` || className}
```

Comment:
You don't have to use the `||` operator. Just add the className along with those default classes, when a user calls the className prop it will override the existing class if it existed in the defaults